samples/bluetooth/mesh/light_ctrl/src/main.c

```diff
@@ -44,10 +48,9 @@ static void isr_emds_cb(void *arg)
 {
 	ARG_UNUSED(arg);
 
-	/* Stop bt and mpsl to reduce power usage and increase storage speed. */
-	(void) bt_disable();
 #if defined(CONFIG_BT_CTLR)
-	mpsl_uninit();
+	/* Stop mpsl to reduce power usage. */
+	mpsl_lib_uninit();
```

Comment:
I still see some spikes in PPK even after calling `mpsl_lib_uninit`. Seems that TASK_CLEAR needs to be triggered for RTC, though not sure if it needs to be added to the sample.

Reply:
Will mpsl_lib_uninit also stop the bluetooth radio?

Reply:
Yes

Reply:
Will this work with NCS 2.1.0?

Reply:
I'm planning to replicate `mpsl_lib_uninit` behavior there with explicit use of `mpsl_uninit` locking RTC0, TIMER0 and RADIO IRQs.